client/dashboard/domains/domain-dns/email-setup.tsx

```diff
@@ -16,8 +18,13 @@ export default function EmailSetup() {
 				<EmailSetupForm
 					label="Google Workspace"
 					key="google-workspace"
-					description={ __(
-						'Paste the verification token provided by Google Workspace for the TXT record.'
+					description={ createInterpolateElement(
+						__(
+							'Paste the verification token provided by Google Workspace for the TXT record. <link />'
```

Comment:
TIL that the default text is `Learn more` haha.

Hmm, maybe we should call the template `<learnMoreLink />`? Not really sure. I believe it will clarify the translators that this sentence is a descriptive text which is followed by a learn more link.

Reply:
Yep, I think that's a good suggestion so I've updated all the `link` elements to `learnMoreLink`